leo/cmd/mod.rs

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
```

Comment:
Please rename all instances of `cmd` and `Cmd` to `command` and `Command` respectively

Reply:
Done!